FirebaseRemoteConfig/SwiftNew/ConfigRealtime.swift

```diff
@@ -568,16 +573,17 @@ class ConfigRealtime: NSObject, URLSessionDataDelegate {
           )
           RCLog.error("I-RCN000021", "Cannot establish connection. Error: \(error)")
           propagateErrors(error)
-          completionHandler(.cancel) // cancel the failing task
         }
       } else {
+        // On success, reset retry parameters.
         remainingRetryCount = maxRetries
         settings.realtimeRetryCount = 0
-        completionHandler(.allow)
       }
+      completionHandler(.allow)
```

Comment:
Looks like the right translation to me, but seems strange that it always calls back with allow.

Reply:
Agreed. Based on https://developer.apple.com/documentation/foundation/nsurlsessiondatadelegate/1410027-urlsession, it looks like it could be reasonable to cancel the request, but I'm maybe slightly leaning to always allowing to avoid a a behavior change.

Reply:
SGTM